AAEmu.Game/Models/Game/Quests/Quest.cs

```
@@ -55,12 +56,17 @@ public uint Start()
                     continue;
                 var acts = QuestManager.Instance.GetActs(component.Id);
                 for (var i = 0; i < acts.Length; i++)
-                    res = acts[i].Use(Owner, this, Objectives[i]);
+                {
+                    if (acts[i].DetailType == "QuestActSupplyItem" & Step == (byte) QuestComponentKind.Supply)
+                        res = acts[i].Use(Owner, this, supplyitem);
```

supplyitem not capitalized -> should throw an error on compile no ?